src/queries.js

```diff
@@ -82,12 +82,28 @@ function queryByText(...args) {
   return firstResultOrNull(queryAllByText, ...args)
 }
 
+function queryAllByTitle(
+  container,
+  text,
+  {exact = true, collapseWhitespace = true, trim = true} = {},
+) {
+  const matcher = exact ? matches : fuzzyMatches
```

Comment:
I'm guessing this is where we're missing some coverage. Could you add one more assertion that specifies `exact: false` to cover this?

Reply:
Sure thing. Will get that pushed up shortly.